packages/blockchain-wallet-v4-frontend/src/modals/Swap/PreviewSwap/PreviewSwapSuccess.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { useEffect, useState } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { Form } from 'redux-form'
+
+import { formatCoin } from '@core/exchange/utils'
+import Remote from '@core/remote'
+import { PaymentValue, SwapNewQuoteStateType } from '@core/types'
+import { Button, HeartbeatLoader, Icon, Link, Text, TextGroup } from 'blockchain-info-components'
+import { ErrorCartridge } from 'components/Cartridge'
+import { FlyoutWrapper, Row, Title, Value } from 'components/Flyout'
+import { GenericNabuErrorFlyout } from 'components/GenericNabuErrorFlyout'
+import { IncomingAmount, InitSwapFormValuesType, SwapAmountFormValues } from 'data/types'
+import { isNabuError } from 'services/errors'
+
+import { QuoteCountDown } from '../../components/QuoteCountDown'
+import { useCountDown } from '../../hooks/useCountDown'
+import { FeeBreakdownBox, FromToLogoLeft, TopText } from '../components'
+import {
+  ExchangeRateRow,
+  IconWrapper,
+  QuoteCountDownWrapper,
+  StyledRow,
+  ToolTipText
+} from './styles'
+
+type Props = {
+  clearErrors: () => void
+  createOrder: () => void
+  error?: any
+  incomingAmount: IncomingAmount
+  initSwapFormValues: InitSwapFormValuesType
+  onClickBack: () => void
+  payment: PaymentValue | undefined
+  quote: SwapNewQuoteStateType
+  returnToInitSwap: () => void
+  submitting: boolean
+  swapAmountFormValues: SwapAmountFormValues
+  trackExchangeTooltip: () => void
+}
+
+export const PreviewSwapSuccess = (props: Props) => {
+  const [isActiveExchangeToolTip, setIsActiveExchangeToolTip] = useState(false)
+  const { isCompletingSoon } = useCountDown(
```

Comment:
Just in case if performance of this step decreases - ideally, we should extract button into a separate component and use the hook there. Otherwise, the whole component is rerendered every second.